src/libs/WorkspacesSettingsUtils.ts

```diff
@@ -125,8 +125,14 @@ function getChatTabBrickRoadReport(policyID?: string): OnyxEntry<Report> {
         return undefined;
     }
 
+    // There are optimistic reports without a reportID, so we need to extract it from the report key
+    const normalizedAllReports = Object.entries(allReports).map(([key, report]) => ({
+        ...report,
+        reportID: report?.reportID ?? key.replace(ONYXKEYS.COLLECTION.REPORT, ''),
+    }));
+
```

Comment:
Same. I think this change is unnecessary if we apply https://github.com/Expensify/App/issues/51488#issuecomment-2443758461. Anyways, could we handle it in https://github.com/Expensify/App/issues/51488 and revert this change? Thanks